analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -22,7 +22,7 @@ package org.ossreviewtoolkit.analyzer
 import java.io.File
 import java.time.Instant
 
-import kotlin.time.measureTimedValue
+import kotlin.time.measureTime
```

Comment:
Hmm, at least to me it's not clear why the concept of introducing IDs generally conflicts with having a composite package curation provider. Esp. as the idea of the composite package curation provider is not actually dropped, but rather inlined, I'd say. I would have appreciated some more details in the commit message here.